### ISSUE TITLE
Question about jsx-child-element-spacing logic and `<br/>` tags

### ISSUE THREAD

**Issue by pascalpp** [yannickcr/eslint-plugin-react#2988]
A couple examples:
```
  return (
    <div className="break-tag-example">
      one line
      <br/>
      another line
    </div>
  );
```

```
  return (
    <div className="break-tag-example">
      one line<br/>
      another line
    </div>
  );
```

Both of these are flagged for ambiguous spacing by jsx-child-element-spacing. But it seems in both cases the intent is clear and both would render as expected in the browser. I wonder if there could be some exception made for `<br/>` tags?

**Comment by ljharb** [comment]
I would definitely expect the first example to be acceptable.

**Comment by pascalpp** [comment]
@ljharb I've opened PR https://github.com/yannickcr/eslint-plugin-react/pull/2989 which simply removes BR from the set of inline elements, because I don't think there's any case where whitespace around a br tag would result in a different result rendered in the browser. Can you have a look?
